### PR TITLE
prometheus: kube-state-metrics

### PR DESCRIPTION
**Pull request by hectorj2f**
This PR is blocked by the chart PR with the respective changes

### CODE REVIEW

**Comment by shaneutt** [comment]
We need  to open this PR against https://github.com/mesosphere/kubernetes-base-addons as well please, as [we're in an interim phase and deprecating this repository](https://mesosphere.slack.com/archives/CQQ4HV5NY/p1575491975235000). In the next few days, we'll update this repo to point out that new PR's only need to be made against the new repo.

File: templates/prometheus.yaml [hunk 1]
```diff
@@ -302,4 +302,4 @@ spec:
           # override the default k8s.gcr.io/kube-state-metrics repositry
           # containerd mirror functionality does not support pulling these images
           # TODO remove once https://github.com/containerd/containerd/issues/3756 is resolved
-          repository: gcr.io/google_containers/kube-state-metrics
+          repository: quay.io/coreos/kube-state-metrics
```
Comment (jimmidyson) [inline templates/prometheus.yaml]: Why has this changed? Sorry I don't have any context to review this PR so sorry for what could be a stupid question.

Comment (hectorj2f) [reply templates/prometheus.yaml]: that is the new image repository used by the official chart to use v1.8.0. That version isn't available for `gcr.io/google_containers/kube-state-metrics`

Comment (hectorj2f) [reply templates/prometheus.yaml]: I am being explicit here about this change

**Pull request merged**